pom.xml

```
@@ -954,7 +954,7 @@
         <plugin>
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-site-plugin</artifactId>
-          <version>3.11.0</version>
+          <version>4.0.0-M4</version>
```

```suggestion
          <version>3.12.1</version>
```
check for availability

---

Confirmed availability